.github/workflows/lint.yml

```diff
@@ -24,9 +24,11 @@ jobs:
             # We are on master, just set the SHA from our current location
             echo ::set-output name=commit_sha::${GITHUB_SHA}
           else
-            # We are on a PR, we need to check out PR branch
-            git checkout ${GITHUB_HEAD_REF}
-            echo ::set-output name=commit_sha::$(git rev-parse ${GITHUB_HEAD_REF})
+            # We are on a PR, so actions/checkout leaves us on merge commit.
+            # Check out the actual tip of the branch.
+            PR_TIP=$(git rev-parse HEAD^2)
```

Comment:
This seems kind of fragile to me though

Reply:
I agree, but it's was what Ed Thompson suggested as the right way to do it—I think it's supposed to be a documented fact that PRs check out to the merge ref (merge A into B), so it *should* be stable rely on the second parent being A.